v2/thirdpartypasswordless/pre-built-ui/setup/frontend.mdx

```diff
@@ -693,14 +701,16 @@ Add the highlighted code snippet to your root level `render` function.
 
 ```tsx
 import React from 'react';
+import { ThirdPartyEmailPasswordPreBuiltUI } from "supertokens-auth-react/recipe/thirdpartyemailpassword/prebuiltui";
 import SuperTokens, { SuperTokensWrapper } from "supertokens-auth-react";
+import { canHandleRoute, getRoutingComponent } from "supertokens-auth-react/ui";
 
 class App extends React.Component {
     render() {
         // highlight-start
-        if (SuperTokens.canHandleRoute()) {
+        if (canHandleRoute([ThirdPartyEmailPasswordPreBuiltUI])) {
             // This renders the login UI on the ^{form_websiteBasePath} route
-            return SuperTokens.getRoutingComponent()
+            return getRoutingComponent([ThirdPartyEmailPasswordPreBuiltUI])
```

Comment:
Wrong recipe. Please make these use the variable `recipePreBuiltUINameCapitalLetters`.